internal/osbuild/copy_stage.go

```diff
@@ -39,6 +39,10 @@ func NewCopyStageSimple(options *CopyStageOptions, inputs Inputs) *Stage {
 	}
 }
 
+type CopyStageFilesInputs map[string]*FilesInput
```

Comment:
There's a `TODO: commit message` for this commit :smile:

Reply:
Can you post the commit SHA? Github is not good at showing that with a review...

Reply:
Got it... https://github.com/osbuild/osbuild-composer/pull/3281/commits/b354e86bbabcbb865a08612401affe12418d941d I'll add a message then :D

Reply:
I'll wait for other reviews before pushing the fixed commit